en/docs/install-and-setup/install-on-vm.md

```diff
@@ -7,55 +7,56 @@ Follow the instructions below to install the WSO2 API Microgateway related binar
 
 ### Microgateway Toolkit
 
-1.  Log in to the command line (Terminal on Mac & Linux).
+1.  Download the [WSO2 API Microgateway toolkit distribution](https://wso2.com/api-management/api-microgateway/).
 
-2.  Download the [WSO2 API Microgateway toolkit distribution](https://wso2.com/api-management/api-microgateway/) .
-
-3.  Extract the archive file to a dedicated directory for the WSO2 API Microgateway **Toolkit** , which will hereafter be referred to as `           <MGW-TK_HOME>          ` .
-
-4.  Append the full path of the `           /bin          ` folder of the extracted WSO2 API Microgateway toolkit distribution to the PATH environment variable.
+2.  Extract the archive file to a dedicated directory for the WSO2 API Microgateway Toolkit, which will hereafter be referred to as `<MGW-TK_HOME>`.
+    
+3.  Append the full path of the `/bin` folder of the extracted WSO2 API Microgateway toolkit distribution to the PATH environment variable.
```

Comment:
better if we don't use folder and directory terms interchangeably in the docs. Shall we use directory?
```suggestion
3.  Append the full path of the `/bin` directory of the extracted WSO2 API Microgateway toolkit distribution to the PATH environment variable.
```

Reply:
Fixed https://github.com/wso2/docs-mg/pull/54/commits/22e1005ff35ccb1e2f1679aeeaba27e6d514bf6c